apps/web/src/constants/poolCandidate.ts

```diff
@@ -43,3 +43,17 @@ export const NOT_PLACED_STATUSES = [
   PoolCandidateStatus.QualifiedAvailable,
   PoolCandidateStatus.PlacedTentative,
 ];
+
+export const RECORD_DECISION_STATUSES = [
+  PoolCandidateStatus.NewApplication,
+  PoolCandidateStatus.ApplicationReview,
+  PoolCandidateStatus.ScreenedIn,
+  PoolCandidateStatus.UnderAssessment,
+];
+
+export const REVERT_DECISION_STATUSES = [
+  PoolCandidateStatus.QualifiedAvailable,
+  PoolCandidateStatus.QualifiedUnavailable,
+  PoolCandidateStatus.QualifiedWithdrew,
+  PoolCandidateStatus.Expired,
+];
```

Comment:
Going to have an issue here, per validation this will fail on half the statuses

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/f43375b2-0eed-4fe2-90c8-78efa707d856)

Which came from #9190

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/3c936f1b-154b-45c3-85f6-56fdc6da3ffe)

Reply:
I'm so confused :face_with_spiral_eyes: So is the AC incorrect? :thinking:

Reply:
oh okay, so just updae the array to match the statuses in the screenshot?

Reply:
Might as well run things by @tristan-orourke to double check 😆

Reply:
Sorry! @vd1992 is correct, the AC are wrong.

Reply:
I'll update them now

Reply:
Actually, why are we adding the revert button in this PR at all? That makes more sense to be done as part of #9197, and the re-instate button added with #9198.

Reply:
okay so... am i just backing out the revert stuff entirely and narrowing the scope to just the record button? :stuck_out_tongue:

Reply:
Uhhh... no, leave it, its already done. Above and beyond the new AC! 😆

Reply:
okay, awesome :+1: 

Updated the statuses in [update revert decision statuses](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10129/commits/d99e18e00409edcacc9f5976b7b8ca94c9ae2ec6)